aggie-coding-club/Rev-Registration
# Automate scrape grades & scrape depts

## Description

This adds a `--recent` flag to both scrape grades and scrape depts, as well as the `--discord` flag for sending success/failure messages to a specified Discord channel. It also adds a workflow with cron schedules for running the jobs on a schedule.

## How to test

There's a few options, but the easiest would probably be to comment out the `on: schedule: ...` part like I did in f108fc7 and force push it to a separate branch then check actions (or the Discord channel) to confirm it works. 

To test the Discord bot locally (just the `--discord` flag), message me for the DISCORD_BOT_TOKEN and DISCORD_SCRAPE_CHANNEL_ID to add to the `.env` file.

## Related tasks

Closes #510, closes #529
